scripts/setUp.ts

```diff
@@ -36,7 +35,6 @@ const setupServer = (process: any) => {
 const server = spawn("npm", ["run", "start"], {});
 
 module.exports = async () => {
-  console.log(`\nSetting up Integration tests...\n\n`);
```

Comment:
Might be okay to leave these logs in as they are only during tests and give us an indication if things fall apart

Reply:
Yup, I will rollback this change.